apps/SageAccountWeb/package.json

```diff
@@ -106,7 +106,7 @@
     ]
   },
   "resolutions": {
-    "@babel/runtime": "7.21.0",
+    "@babel/runtime": "7.20.13",
```

Comment:
I keep running into weird local-only build issues with create-react-app trying to use the wrong version of this package. Changing it makes the issue go away, but eventually it decides to use a different version and breaks again.